src/spatial_ref/srs.rs

```diff
@@ -451,6 +451,141 @@ impl SpatialRef {
             None
         }
     }
+
+    /// Get spheroid semi major axis.
+    ///
+    /// Returns [`Err`] variant in case of non [`OGRERR_NONE`](https://gdal.org/api/vector_c_api.html#c.OGRERR_NONE) error result from the C library.
```

Comment:
```suggestion
    /// Returns [`Err`] variant in case the C library returns an error code different from [`OGRERR_NONE`](https://gdal.org/api/vector_c_api.html#c.OGRERR_NONE).
```